charts/base-cluster/values.schema.json

```diff
@@ -1263,6 +1263,81 @@
       },
       "additionalProperties": false
     },
+    "weaveworks": {
+      "type": "object",
+      "description": "Values for the gitops-chart",
+      "properties": {
+        "enabled": {
+          "type": "boolean"
+        },
+        "adminUser": {
+          "type": "object",
+          "properties": {
+            "create": {
+              "type": "boolean"
+            },
+            "username": {
+              "type": "string"
+            },
+            "password": {
+              "type": "string"
+            }
+          },
+          "if": {
+            "properties": {
+              "create": {
+                "const": true
+              }
+            },
+            "required": ["create"]
+          },
+          "then": {
+            "required": ["username","password"]
+          },
+          "additionalProperties": false
+        },
+        "ingress": {
+          "type": "object",
+          "properties": {
+            "enabled": {
+              "type": "boolean"
+            },
+            "host": {
+              "type": "string"
+            }
+          },
+          "additionalProperties": false
+        },
+        "oidcSecret": {
+          "type": "object",
+          "properties": {
+            "create": {
```

Comment:
Kinda the same thing here, there is no need for a `create` parameter, we will connect the OIDC if `global.authentication` is configured, just like with grafana

Reply:
oidc-configuration removed. It will be enabled, if global.authentication is enabled